modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/PodcastSyncProcess.kt

```diff
@@ -931,44 +937,85 @@ class PodcastSyncProcess(
             bookmarkManager.upsertSynced(bookmark.copy(syncStatus = SyncStatus.SYNCED))
         }
     }
-}
 
-private fun toRecord(podcast: Podcast): Record =
-    record {
-        this.podcast = syncUserPodcast {
-            podcast.addedDate?.toInstant()?.epochSecond?.let { epochSecond ->
-                dateAdded = timestamp { seconds = epochSecond }
-            }
+    companion object {
 
-            folderUuid = stringValue {
-                val folderUuid = podcast.folderUuid
-                value = if (folderUuid.isNullOrEmpty()) {
-                    Folder.homeFolderUuid
-                } else {
-                    folderUuid
+        @VisibleForTesting
+        internal fun toRecord(podcast: Podcast): Record =
+            record {
+                this.podcast = syncUserPodcast {
+                    podcast.addedDate?.toInstant()?.epochSecond?.let { epochSecond ->
+                        dateAdded = timestamp { seconds = epochSecond }
+                    }
+
+                    folderUuid = stringValue {
+                        val folderUuid = podcast.folderUuid
+                        value = if (folderUuid.isNullOrEmpty()) {
+                            Folder.homeFolderUuid
+                        } else {
+                            folderUuid
+                        }
+                    }
+
+                    isDeleted = boolValue { value = !podcast.isSubscribed }
+                    subscribed = boolValue { value = podcast.isSubscribed }
```

Comment:
This does seem a bit weird, so I double-checked and confirmed it's also what [iOS is doing](https://github.com/Automattic/pocket-casts-ios/blob/a081912bba9b143a547c4023175f3d5e643c336f/Modules/Server/Sources/PocketCastsServer/Public/Sync/SyncTask%2BLocalChanges.swift#L17-L18):

``` swift
podcastRecord.isDeleted.value = !podcast.isSubscribed()
podcastRecord.subscribed.value = podcast.isSubscribed()
```

Reply:
A comment here is a good idea. I've asked for some clarification around why both values exist (p1705498431040889-slack-C029BMW150R), and I'll add a comment in a separate PR when I get that clarification.